docs/clusterdefinition.md

```diff
@@ -134,6 +135,54 @@ Additionally above, we specified a custom docker image for tiller, let's say we
 
 Finally, the `addons.enabled` boolean property was omitted above; that's by design. If you specify a `containers` configuration, acs-engine assumes you're enabling the addon. The very first example above demonstrates a simple "enable this addon with default configuration" declaration.
 
+#### kubeletConfig
+
+`kubeletConfig` declares runtime configuration for the kubelet running on all master and agent nodes. It is a generic key/value object, and a child property of `kubernetesConfig`. An example custom kubelet config:
+
+```
+"kubernetesConfig": {
+    "kubeletConfig": {
+        "--eviction-hard": "memory.available<250Mi,nodefs.available<20%,nodefs.inodesFree<10%"
+    }
+}
+```
+
+See [here](https://kubernetes.io/docs/reference/generated/kubelet/) for a reference of supported kubelet options.
+
+Below is a list of kubelet options that acs-engine will configure by default:
+
+|kubelet option|default value|
+|---|---|
+|"--pod-infra-container-image"|"pause-amd64:<version>"|
+|"--max-pods"|"110"|
+|"--eviction-hard"|"memory.available<100Mi,nodefs.available<10%,nodefs.inodesFree<5%"|
+|"--node-status-update-frequency"|"10s"|
+|"--image-gc-high-threshold"|"85"|
+|"--image-gc-low-threshold"|"850"|
+|"--non-masquerade-cidr"|"10.0.0.0/8"|
+
+Below is a list of kubelet options that are *not* currently user-configurable, either because a higher order configuration vector is available that enforces kubelet configuration, or because a static configuration is required to build a functional cluster:
+
+|kubelet option|default value|
+|---|---|
+|"--address"|"0.0.0.0"|
+|"--azure-container-registry-config"|"/etc/kubernetes/azure.json"|
+|"--allow-privileged"|"true"|
+|"--pod-manifest-path"|"/etc/kubernetes/manifests"|
+|"--cluster-domain"|"cluster.local"|
+|"--cloud-config"|"/etc/kubernetes/azure.json"|
+|"--cloud-provider"|"azure"|

```

Comment:
This can move up to configurable

Reply:
good catch!

Reply:
Oh, actually: "either because a higher order configuration vector is available that enforces kubelet configuration". In this case we're calling it static because it is overridable by useCloudControllerManager. So I think we want to keep it here in terms of documentation (in other words, we don't want to encourage folks to modify `--cloud-provider`, we want them instead to set `useCloudControllerManager` to true to bypass azure cloud-provider implementation.